src/Passes/Source/Rules/Notation/Select.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Rules/Notation/Notation.hpp"
+#include "Rules/Operands/Any.hpp"
+#include "Rules/Operands/Call.hpp"
+#include "Rules/Operands/Instruction.hpp"
+
+#include "Llvm/Llvm.hpp"
+
+#include <unordered_map>
+#include <vector>
+
+namespace microsoft
+{
+namespace quantum
+{
+    namespace notation
```

Comment:
So far (in qsharp-runtime) all the namespace names were capitalized (**M**icrosoft, **Q**uantum).

If this is C++17 or newer then I'd suggest
```suggestion
namespace Microsoft::Quantum::Notation
{
```